src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/AutoInsert/OnAutoInsertEndpoint.cs

```diff
@@ -47,7 +47,7 @@ internal class OnAutoInsertEndpoint : AbstractRazorDelegatingEndpoint<OnAutoInse
 
         protected override string CustomMessageTarget => RazorLanguageServerCustomMessageTargets.RazorOnAutoInsertEndpointName;
 
-        public RegistrationExtensionResult GetRegistration(VSInternalClientCapabilities clientCapabilities)
+        public RegistrationExtensionResult? GetRegistration(VSInternalClientCapabilities clientCapabilities)
```

Comment:
Please don't make this null-able unless there's a need. I've had a lot of trouble with Razor-CLaSP related to nullables on AbstractRazorDelegatingEndpoint.